index.js

```diff
@@ -22,12 +22,28 @@ const SUPPORTED_GRAMMARS = [
 	'source.js.jsx'
 ];
 
+let currentLine = null;
+let currentChar = null;
+
+const goToError = () => {
+	const editor = atom.workspace.getActiveTextEditor();
+
+	if (!editor || !currentLine || !currentChar) {
+		return;
+	}
+
+	editor.setCursorBufferPosition([currentLine - 1, currentChar - 1]);
+};
+
 const jsHintStatusBar = document.createElement('span');
 jsHintStatusBar.setAttribute('id', 'jshint-statusbar');
 jsHintStatusBar.classList.add('inline-block');
+jsHintStatusBar.addEventListener('click', goToError);
```

Comment:
I think it would be more obvious that it's clickable if you make the text there a link so it get's a underline on hover. Similar to the `UTF-8` link in the statusbar.